### ISSUE TITLE
일본어를 지원할 예정은 없나요?

### ISSUE THREAD

**Issue by 367834586** [khris/naver-endic-unofficial#29]
파이어폭스에서 유일하게 간편하게 사용할수 있는 사전 플러그인인데, 영어만 지원한다는게 너무 아쉬워요 
일본어를 지원할 예정은 없나요?

**Comment by khris** [comment]
네. 당장은 제 사정도 사정이고(좀 바빠서...) 일본어를 지원할 생각은 없습니다. 아마 제가 다시 작업에 들어가도 밀린 이슈나 기능개선을 할 생각이라서요... 다만 소스코드를 수정해서 빌드하실 수 있으시면 https://github.com/khris/naver-endic-unofficial/blob/master/load_dict.js#L25-L26 이 부분을 수정하시면 될거같습니다.

추후에는 chardet등을 이용해서 선택한 단어가 어느 언어인지 판별한 후 적당한 사전을 보여주려고 하지만, 그게 언제가 될지는 잘 모르겠습니다 ^^;
